src/main/cliapp/src/containers/controlledVocabularyPage/ControlledVocabularyComponent.js

```diff
@@ -392,7 +393,7 @@ export const ControlledVocabularyComponent = () => {
           key={col.field}
           field={col.field}
           header={col.header}
-          sortable={isEnabled}
+          sortable={(col.field == "id")? false : isEnabled}
```

Comment:
Maybe sortable on the <Column> component should just be set to col.sortable and set the sortable field on the ID column object to false?

Reply:
Not sure I understand this comment.

Reply:
Please review the File changed now.

Reply:
I think we just said the same thing. I just didn't notice your comment.